### PR TITLE
ACS-6309 fix reportportal dependencies

### PR DESCRIPTION
**Pull request by wojtekswieton**

### CODE REVIEW

File: tests/tas-cmis/pom.xml [hunk 1]
```diff
@@ -67,6 +67,12 @@
             <classifier>tests</classifier>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.epam.reportportal</groupId>
+            <artifactId>agent-java-testng</artifactId>
+            <version>5.4.0</version>
```
Comment (dsibilio) [inline tests/tas-cmis/pom.xml:73]: The version should be unnecessary, it should already be handled via the `dependencyManagement` of the parent POM. Can we test removing it?

File: tests/tas-email/pom.xml [hunk 2]
```diff
@@ -33,6 +33,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.epam.reportportal</groupId>
+            <artifactId>agent-java-testng</artifactId>
+            <version>5.4.0</version>
```
Comment (dsibilio) [inline tests/tas-email/pom.xml:39]: Same as above.

File: tests/tas-integration/pom.xml [hunk 3]
```diff
@@ -34,6 +34,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.epam.reportportal</groupId>
+            <artifactId>agent-java-testng</artifactId>
+            <version>5.4.0</version>
```
Comment (dsibilio) [inline tests/tas-integration/pom.xml:40]: Same as above.

File: tests/tas-restapi/pom.xml [hunk 4]
```diff
@@ -79,6 +79,12 @@
             <version>${dependency.awaitility.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.epam.reportportal</groupId>
+            <artifactId>agent-java-testng</artifactId>
+            <version>5.4.0</version>
```
Comment (dsibilio) [inline tests/tas-restapi/pom.xml:85]: Same as above.

File: tests/tas-webdav/pom.xml [hunk 5]
```diff
@@ -37,6 +37,12 @@
             <classifier>tests</classifier>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.epam.reportportal</groupId>
+            <artifactId>agent-java-testng</artifactId>
+            <version>5.4.0</version>
```
Comment (dsibilio) [inline tests/tas-webdav/pom.xml:43]: Same as above.

**Review by dsibilio — CHANGES_REQUESTED** [review]

File: tests/tas-cmis/pom.xml (hunk 1 above)
Comment (dsibilio) [inline tests/tas-cmis/pom.xml:73]:
```suggestion
```

File: tests/tas-email/pom.xml (hunk 2 above)
Comment (dsibilio) [inline tests/tas-email/pom.xml:39]:
```suggestion
```

File: tests/tas-integration/pom.xml (hunk 3 above)
Comment (dsibilio) [inline tests/tas-integration/pom.xml:40]:
```suggestion
```

File: tests/tas-restapi/pom.xml (hunk 4 above)
Comment (dsibilio) [inline tests/tas-restapi/pom.xml:85]:
```suggestion
```

File: tests/tas-webdav/pom.xml (hunk 5 above)
Comment (dsibilio) [inline tests/tas-webdav/pom.xml:43]:
```suggestion
```

**Review by dsibilio — CHANGES_REQUESTED** [review]

**Review by dsibilio — APPROVED** [review]

**Review by aonych — APPROVED** [review]

**Pull request merged**